packages/core/assemblyManager/assembly.ts

```diff
@@ -228,7 +233,7 @@ export default function assemblyFactory(assemblyConfigType: IAnyType) {
           // eslint-disable-next-line @typescript-eslint/no-explicit-any
           self as any,
           // @ts-ignore
-          loadAssemblyData,
+          makeLoadAssemblyData(pluginManager),
```

Comment:
this may be creating a closure over pluginManager. I dunno @rbuels you have sometimes said things like "avoid closing over blah" before, I don't know a lot about that